.github/workflows/lambdas.yml

```diff
@@ -0,0 +1,90 @@
+on:
+    push:
+      paths:
+        - '.github/workflows/lambdas.yml'
+        - 'lambdas/**'
+jobs:
+  linter:
```

Comment:
Instead of copy-paste you might want this: https://docs.github.com/en/actions/using-workflows/reusing-workflows